rtv/submission_page.py

```diff
@@ -138,18 +139,59 @@ def prompt_subreddit(self):
     @SubmissionController.register(Command('SUBMISSION_OPEN_IN_BROWSER'))
     def open_link(self):
         """
-        Open the selected item with the web browser
+        Open the selected link
+
+        Prompt user to choose which link to open if additional links
+        are mentioned at the item.
         """
 
         data = self.get_selected_item()
         if data['type'] == 'Submission':
-            self.term.open_link(data['url_full'])
+            self.open_submission_or_permalink_or_mentioned_link(data)
             self.config.history.add(data['url_full'])
```

Comment:
I don't really understand what the history is used for, because I found no documentation on it. Nevertheless I implemented what you described in 5fcd7e517a58539e94f9165eca16ff555b34e6a5.

Reply:
fyi history is used to check if a link should be colored blue or purple to have the effect of links changing color when you click on them.